### ISSUE TITLE
V2.4.7 password meter - usability requirement?

### ISSUE THREAD

**Issue by elarlang** [OWASP/ASVS#420]
"Verify that a password strength meter is provided to help users set a stronger secret." - feels more like usability requirement. All actual requirements are defined in separate points (2.4.1 .. 2.4.4) already.

... and if we want to go with requirements to "educate users" field, then the must have requirement text there should be: "use unique password which you don't use anywhere else" to address password re-use problem.

**Comment by ronperris** [comment]
@elarlang Have you had a peak at [NIST Special Publication 800-63B](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63b.pdf)?

From the document:
> Verifiers SHOULD offer guidance to the subscriber, such as a password-strength meter [Meters], to assist the user in choosing a strong memorized secret. 

The NIST cites [this research](https://www.ndss-symposium.org/wp-content/uploads/2017/09/06_3_1.pdf), which states, "Recent studies have found evidence that some meters actually guide users to choose better passwords—which is a rare bit of good news in password research."

[This](https://www.usenix.org/system/files/conference/usenixsecurity12/sec12-final209.pdf) paper concludes that, "We have conducted the first large-scale study of password-strength meters, finding that meters did affect user behavior and security." based on a "2,931-subject study of 14 password meters".

Has there been some new research that invalidates these studies?

**Comment by m8urnett** [comment]
@elarlang fwiw, I agree. Not that password meters are bad, but that most of the time they are done wrong: My 30-character passwords shows as weak but add a number at the end and then I'm off the scale!

Since ASVS doesn't really work great with SHOULD items, it seems like it would be better left off.

**Comment by jmanico** [comment]
There is clear research showing that password meters help users pick good passwords. If you want to help reword this to make it the RIGHT password meter please open  a new issue but I am going to close this now. Stated with respect for Mark and Elarlang's comments.
